plugins/parsers/influx/machine.go

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
this file is generated and should not be modified. update machine.go.rl instead.

Reply:
Will fix. Seems like my editor was of the opinion running `go fmt` on it would be a good idea... :-)